platform/ios/MGLMapboxEvents.m

```diff
@@ -144,6 +142,8 @@ @interface MGLMapboxEvents () <CLLocationManagerDelegate>
 @property (atomic) NSString *appName;
 @property (atomic) NSString *appVersion;
 @property (atomic) NSString *appBuildNumber;
+@property (atomic) NSNumber *version;
```

Comment:
This should just be a global `const NSUInteger`, since there’s no reason for subclassers to override the value or for the value to change at runtime.